src/SessionSetup/ConnectToStadiaWidget.ui

```diff
@@ -149,6 +149,75 @@
              <property name="spacing">
               <number>7</number>
              </property>
+             <item>
+              <layout class="QHBoxLayout" name="instanceSettingsHBoxLayout" stretch="0,0,1,0,0">
+               <item>
+                <widget class="QComboBox" name="comboBox"/>
+               </item>
+               <item>
+                <spacer name="horizontalSpacer_2">
+                 <property name="orientation">
+                  <enum>Qt::Horizontal</enum>
+                 </property>
+                 <property name="sizeHint" stdset="0">
+                  <size>
+                   <width>40</width>
+                   <height>20</height>
+                  </size>
+                 </property>
+                </spacer>
+               </item>
+               <item>
+                <widget class="QLineEdit" name="instancesFilterLineEdit">
+                 <property name="placeholderText">
+                  <string>Filter</string>
+                 </property>
+                </widget>
+               </item>
+               <item>
+                <widget class="QCheckBox" name="alInstanceslCheckBox">
```

Comment:
The name looks like a typo, probably should be `allInstancesCheckBox`?

Reply:
yes, done